### ISSUE TITLE
Parallelize test runners

### ISSUE THREAD

**Issue by s1na** [ethereumjs/ethereumjs-monorepo#535]
Almost all of the test cases are independent of each other and can be very well run in parallel. This should reduce the time for running the whole test suite significantly and improve developer experience.

**Comment by holgerd77** [comment]
This has partly been done on the CI level by splitting up the tests and generally there has been a lot of work being done by Everton to get to descent CI times. Will close here for now, issue is also somewhat old and pretty short regarding the initial comment description.

If someone sees the need in the future feel free to reopen or create a new issue.
